llvm/test/CodeGen/ARM/big-endian-vmov.ll

```diff
@@ -10,7 +10,7 @@ define arm_aapcs_vfpcc <8 x i8> @vmov_i8() {
 ;
 ; CHECK-BE-LABEL: vmov_i8:
 ; CHECK-BE:       @ %bb.0:
-; CHECK-BE-NEXT:    vmov.i64 d0, #0xff
+; CHECK-BE-NEXT:    vmov.i64 d0, #0xff00000000000000Î
```

Comment:
I believe these ones were correct before, and either need a `vmov 0xff`  or `vmov 0xff00000000000000 + vrev`